core/commands/name/name.go

```diff
@@ -84,27 +79,25 @@ Resolve the value of a dnslink:
 }
 
 type IpnsInspectValidation struct {
-	Valid     bool
-	Reason    string
-	PublicKey peer.ID
+	Valid  bool
+	Reason string
+	Name   string
 }
 
 // IpnsInspectEntry contains the deserialized values from an IPNS Entry:
 // https://github.com/ipfs/specs/blob/main/ipns/IPNS.md#record-serialization-format
 type IpnsInspectEntry struct {
-	Value        string
-	ValidityType *ipns_pb.IpnsEntry_ValidityType
+	Value        *path.Path
+	ValidityType *ipns.ValidityType
 	Validity     *time.Time
-	Sequence     uint64
-	TTL          *uint64
-	PublicKey    string
-	SignatureV1  string
-	SignatureV2  string
-	Data         interface{}
+	Sequence     *uint64
+	TTL          *time.Duration
 }
 
 type IpnsInspectResult struct {
 	Entry      IpnsInspectEntry
+	Version    string
```

Comment:
Two asks here:

1.  This is present in `--enc=json`  output, but missing from default text one

2. Version may be confused with Sequence. Let' make it less ambigous, what matters is "signature type" used for validation, so maybe:
```suggestion
	SignatureType    string
```

This way we have "V1+V2" and "V2" as distinct signature types.